cmd/fs-v1-metadata.go

```diff
@@ -269,6 +284,12 @@ func loadFormatFS(fsPath string) (*formatConfigV1, error) {
 		return nil, err
 	}
 
+	// Check if format is V2.
+	if !isFSFormatV2(format) {
```

Comment:
I think it is important if we return two errors when we check for FS format version:
- Old version so we can migrate
- Newer unsupported version (like, 3, 4, ..), so we stop here since things start to be unpredictable.